paper-dropdown.js

```diff
@@ -303,6 +332,9 @@ class PaperDropDown extends Polymer.mixinBehaviors([PaperDropdownBehavior], Poly
                 :host * {
                     outline: none;
                 }
+                paper-dropdown-menu{
+                    display: block;
+                }
```

Comment:
`paper-dropdown` is supposed to be just a wrapper around `paper-dropdown-menu` and hence should not add any custom css to it. If need arises, user can use `--paper-dropdown-menu` css mixin to apply their custom css.